dataactbroker/handlers/domainHandler.py

```diff
@@ -0,0 +1,28 @@
+from dataactcore.utils.jsonResponse import JsonResponse
+from dataactcore.utils.statusCode import StatusCode
+
+
+class DomainHandler:
+
+    def __init__(self, interfaces=None):
+        if interfaces is not None:
+            self.interfaces = interfaces
+            self.validationManager = interfaces.validationDb
+
+    def addInterfaces(self, interfaces):
+        """ Add connections to databases
+
+        Args:
+            interfaces: InterfaceHolder object to DBs
+        """
+        self.interfaces = interfaces
+        self.validationManager = interfaces.validationDb
+
+    def listAgencies(self):
+        agencies = self.validationManager.getAllAgencies()
+        agency_list = []
+
+        for agency in agencies:
```

Comment:
Relevant to the comments re: surrogate keys, I'm used to seeing a pattern of sending an id/name to the front end, and then the front-end returns the id for us to stick in the db. Doe you think we should send the cgac_id too?